### PR TITLE
Bug 1160491 - Add a Build Flag to create a setting for Spark Dogfooders.

### PR DESCRIPTION
**Pull request by tamarahills**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8ece31b05d7fd129b291cc8ab59b6a62c7822a6c

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b616956afe269b64d9e23f476ddfc43aba8b2c7e

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=357b4f16c5d697fbfdf3bed6839871ec4eb245c1

File: build/settings.js [hunk 1]
```diff
@@ -282,6 +282,11 @@ function execute(config) {
     settings['devtools.qps.enabled'] = true;
   }
 
+  if (config.DOGFOOD === '1') {
+    settings['debug.performance_data.spark_dogfooding'] = '1';
```
Comment (DouglasSherk) [inline build/settings.js]: I think we should generally avoid conflating dogfooding and Spark. They support each other, but they aren't the same thing. Thus I would suggest a name 'debug.performance_data.dogfooding' instead.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d705f676783c27c119886e67cd5df8bcc5c6c35f

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0f9eca54d3980dae6ae2164203c18d60259f1557

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=14100c3b726a356bc53baf38d550b90deb08c394

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e78899e95d74463aa90d094ea43f272bb1d4d3b7

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c5b108c8521cbcfe81787c9817580d71a0a9f75c

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=22ae16db0c84a4c4286bc40c302324ca7e10edef

**Pull request merged**